packages/alias/test/test.js

```diff
@@ -10,6 +10,47 @@ import alias from '../dist';
 const normalizePath = (pathToNormalize) => slash(pathToNormalize.replace(/^([A-Z]:)/, ''));
 const DIRNAME = normalizePath(__dirname);
 
+// "plugins" is an array of plugins for Rollup, they should include "alias"
```

Comment:
could you convert those 2 lines to a JSDOC? that way this function would get nice hints in IDEs